### ISSUE TITLE
Redundant umbilical skin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7460]
**Preferred term label:**
Redundant umbilical skin

**Synonyms**

**Definition (free text, please give PubMed ID)**
Greater than normal amount of skin surrounding the umbilicus (belly button) with protrusion of the umbilicus above the plane of the abdomen.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**OMIM:
614557 
PMID:24773188  1/1, neonatal
**Your nano-attribution (ORCID)**

**Comment by umberagarwal** [comment]
I feel that it will be rather hard to be certain or suspicious on fetal scan about redundant umbilical skin. I wonder if 'excessive' umbilical or 'excessively redundant' umbilical skin would be more appropriate. One will need a good sagittal plane of anterior abdominal wall at level of cord insertion. An umbilical hernia can distend the umbilical ring and push it out and may give a false impression of excessive umbilical skin. I presume that this is a diagnosis once umbilical hernia or exomphalos has been excluded. 

Diseases characterized by this term:
Morquio syndrome- Omim 252300
FKBP14-related Ehlers-Danlos syndrome (EDS)- Omim # 614557
Axenfeld-Rieger syndrome (ARS) - there are 3 omim entries- type 1 #180050; type 2 %601499; type 3 #602482

https://orcid.org/my-orcid?orcid=0000-0001-7600-5359

**Comment by pnrobinson** [comment]
This feature can also be seen postnatally, see 
Axenfeld-Rieger syndrome, type 1 (OMIM:180500 )
FIgure 4c https://pubmed.ncbi.nlm.nih.gov/8944018/

**Comment by pnrobinson** [comment]
added
